app/src/main/res/values/example_descriptions.xml

```diff
@@ -86,4 +86,5 @@
     <string name="description_3d_globe">Showcasing 3D globe</string>
     <string name="description_osm">Using raster source and layer to render OSM tiles with TileSet class</string>
     <string name="description_view_annotation">Callout view anchored to the marker</string>
```

Comment:
Maybe the "advanced" example could be renamed to something else, like: `Add view annotation anchored to a symbol layer feature`

Reply:
nice, will apply your wording!